MIPs/new-protocol_metadata.md

```diff
@@ -0,0 +1,159 @@
+---
+mip: 
+title: Protocol Metadata
+author: Byron Blattel (@alchebyte),  Wigy (@wigy-opensource-developer), Bartmoss (@izolyomi)
+status: Draft
+type: Standards
+category: Protocol
+created: 
+---
+# MIP Process and Guidelines
```

Comment:
Copy-paste error for title, still noone noticed. 😆

Reply:
fixed :)